keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -40,7 +42,7 @@ fun ClusterSpec.resolve(): Set<ServerGroup> =
       health = resolveHealth(it.name),
       scaling = resolveScaling(it.name),
       tags = defaults.tags + overrides[it.name]?.tags,
-      deliveryArtifact = deliveryArtifact,
+      artifactName = artifactName,
```

Comment:
Artifact properties have been flattened out into discrete fields as per `ArtifactProvider` and `ArtifactReferenceProvider`.